docs/rule/no-action-modifiers.md

```diff
@@ -2,6 +2,8 @@
 
 This rule forbids the use of `{{action}}` modifiers on elements.
 
+In Ember 3.11+, the recommended alternative is the `on` modifier. Prior to Ember 3.11, the alternative is to use HTML element event properties such as `onclick`.
```

Comment:
Maybe we should just suggest `{{on` all the time? [ember-on-modifier](https://github.com/buschtoens/ember-on-modifier) is tested all the way back to 2.13 (uses the `ember-modifier-manager-polyfill` for older version support), what do you think?

Reply:
Agreed, updated to only suggest `on` and link to the polyfill.